src/react/src/components/ClaimFacilityStepper.jsx

```diff
@@ -117,23 +121,23 @@ const SUBMIT_FORM = 'SUBMIT_FORM';
 
 const steps = Object.freeze([
     Object.freeze({
-        name: 'Claim this facility',
+        name: facilityClaimStepsNames.CLAIM_PROD_LOCATION,
         component: ClaimFacilityIntroStep,
-        next: 'Support Documentation',
+        next: facilityClaimStepsNames.SUPPORT_DOC,
         hasBackButton: true,
         hasNextButton: true,
-        stepInputIsValid: stubTrue,
+        stepInputIsValid: claimFacilityIntroIsValid,
```

Comment:
you can check **agreement** here (boolean result) instead of using function **claimFacilityIntroIsValid**

Reply:
If you want to check the validity of the value, you should create a function that destructures the object, retrieves the `agreement` value, and then returns a boolean indicating its validity, rather than returning the value itself. I think it's better to have a separate function for this purpose rather than doing it inline.

Do you only need to check if it is not empty? If so, then this solution can be useful.
E.g. 
`({ agreement }) => !!agreement`

Reply:
@Innavin369 ^^^